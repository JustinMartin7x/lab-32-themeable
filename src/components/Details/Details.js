import React from 'react';
import PropTypes from 'prop-types';

const Details = ({ image }) => (
  <div>
    <img src={image} alt="character image" />
  </div>
);

Details.propTypes = {
  image: PropTypes.string,
};
export default Details;
