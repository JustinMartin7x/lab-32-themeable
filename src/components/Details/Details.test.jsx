import React from 'react';
import { cleanup, render, screen, waitFor } from '@testing-library/react';
import Details from './Details';


describe('renders Detail Page', () => {
  afterEach(() => cleanup());

  it ('will render a Detail page that consists of an image', () => {
    const { asFragment } =
    render(<Details 
      image = "some text"

    />);

    expect(asFragment()).toMatchSnapshot();

  });
});

