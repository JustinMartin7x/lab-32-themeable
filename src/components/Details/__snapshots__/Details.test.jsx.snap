// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Detail Page will render a Detail page that consists of an image 1`] = `
<DocumentFragment>
  <div>
    <img
      alt="character image"
      src="some text"
    />
  </div>
</DocumentFragment>
`;
